README.md

```diff
@@ -64,6 +64,7 @@ Type w*h | Scan Multiplexing | Program commandline flags    | Remark
 64x32 |  1:8              | --led-rows=32 --led-cols=64 --led-multiplexing=1 | few mux choices
 32x32 |  1:16             | --led-rows=32                |
 32x32 |  1:8              | --led-rows=32 --led-multiplexing=1 | few mux choices
+32x32 |  1:8              | --led-rows=32 --led-cols=32 --led-multiplexing=5 | Outdoor Panel 
```

Comment:
I guess we don't need this line, as the above (mentioning "_few mux choices_") already covers this.